tests/v2api.test.ts

```diff
@@ -57,4 +59,26 @@ describe.runIf(semver.gte(nodeVersion, '18.13.0'))('V2 functions API', () => {
       expect(statusCode).toBe(200)
     },
   )
+
+  testMany(
+    'Handles a basic TypeScript function',
+    ['todo:bundler_default', 'bundler_esbuild', 'bundler_esbuild_zisi', 'todo:bundler_default_nft', 'todo:bundler_nft'],
+    async (options) => {
+      const { files, tmpDir } = await zipFixture('v2-api-ts', {
+        opts: merge(options, {
+          archiveFormat: ARCHIVE_FORMAT.NONE,
+          featureFlags: { zisi_pure_esm: true, zisi_functions_api_v2: true },
+        }),
+      })
+
+      const [{ name: archive, entryFilename }] = files
+      const func = await importFunctionFile(`${tmpDir}/${archive}/${entryFilename}`)
```

Comment:
wait - i'm unsure if it's actually transpiling to TS or, if Vitest magic is hiding that. i'll check that first